package-lock.json

```diff
@@ -0,0 +1,15331 @@
+{
```

Comment:
There should not be a package-lock.json for the app nor the e2e testing dir. When you download dependencies, make sure to download using `yarn install`. Having both a package-lock and yarn-lock is not a good idea and will create confusion.